ansible/roles/dashmate/tasks/main.yml

```diff
@@ -269,6 +269,35 @@ Dashmate Config Changed: {{ is_dashmate_config_changed }}\n"
     chdir: '{{ dashmate_cwd }}'
   changed_when: build_dashmate_helper.rc == 0
 
+- name: Get service statuses
+  ansible.builtin.command: "{{ dashmate_cmd }} status services --format=json"
+  become: true
+  become_user: dashmate
+  changed_when: started_dashmate_services.rc == 0
+  args:
+    chdir: '{{ dashmate_cwd }}'
+  register: started_dashmate_services
+  when: mode == 'seed'
+
+- name: Set facts
+  ansible.builtin.set_fact:
+    containers_to_stop: >-
+      {{
+        started_dashmate_services.stdout |
+        from_json |
+        json_query('[?service!=`dashmate_helper`&&service!=`core`&&service!=`drive_tenderdash`]') |
+        map(attribute='containerId') |
+        list
+      }}
+  when: mode == 'seed'
+
+- name: Stop some services in seed mode
+  community.docker.docker_container:
+    name: "{{ item }}"
+    state: stopped
+  when: mode == 'seed'
+  loop: "{{ containers_to_stop }}"
+
```

Comment:
What are we doing here? Task names are not representative. Please rename tasks and add comments.